src/menus/PluginMenus.cpp

```diff
@@ -385,7 +384,6 @@ BaseItemSharedPtr EffectMenu()
          { return Items( wxEmptyString, MenuHelper::PopulateEffectsMenu(
             EffectTypeProcess,
             AudioIONotBusyFlag() | TimeSelectedFlag() | WaveTracksSelectedFlag(),
-            IsRealtimeNotActiveFlag(),
```

Comment:
The definition of `IsRealtimeNotActiveFlag` is in this file.  It becomes unused, so may be deleted too.